tests/unit/api/document/test_document_tables_api.py

```diff
@@ -33,7 +33,7 @@ def test_post(self, get_proxy: MagicMock, RequestParser: MagicMock) -> None:
     @patch('search_service.api.document.get_proxy_client')
     def test_put(self, get_proxy: MagicMock, RequestParser: MagicMock) -> None:
         mock_proxy = get_proxy.return_value = Mock()
-        RequestParser().parse_args.return_value = dict(data='{}', index='fake_index')
+        RequestParser().parse_args.return_value = dict(data=[], index='fake_index')
```

Comment:
could you add a unit test to update multiple values as it is broken before?

Reply:
Sounds good, will do.